internal/init/init.go

```diff
@@ -156,3 +166,22 @@ func writeVscodeConfig(fsys afero.Fs) error {
 	fmt.Println("Generated VS Code settings in " + utils.Bold(settingsPath) + ". Please install the recommended extension!")
 	return nil
 }
+
+func writeIntelliJConfig(fsys afero.Fs) error {
+	if err := utils.MkdirIfNotExistFS(fsys, intellijDir); err != nil {
+		return err
+	}
+	if err := createDenoConfig(denoPath, intelliJDeno, fsys); err != nil {
+		return err
+	}
+
```

Comment:
We can replace this block with `utils.WriteFile`

Reply:
Done. good suggestion thanks.